FogCreek/Glitch-Community
# Peppered fahrenheit - Add Delete button to Project Pages

## Links
* Remix link: https://peppered-fahrenheit.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3328419/

## GIF/Screenshots:
![delete-project](https://user-images.githubusercontent.com/519336/56678667-41b23580-6691-11e9-9327-0b1fc5a71183.gif)
Delete Project button launches pop-over with notice that user can always undelete a project from their profile page

![image](https://user-images.githubusercontent.com/519336/56678686-4e368e00-6691-11e9-90ab-0a7dc5f0d9c8.png)
Also updated the delete collection button to match styling

## Changes:
* Delete Project button is added to the project page underneath project Read.Me.
* In lieu of having a pop-over Component, I reused the existing `popover-with-button` pop-over.  Updated to use the Button and Emoji components as necessary.

## How To Test:
* Try deleting one of your projects from a project page.  Deleting should redirect you to your profile page.
* You should only be able to delete projects you're a member of.
